setupext.py

```diff
@@ -1153,8 +1147,17 @@ def do_custom_build(self):
                 cmd.write(prepare_build_cmd(FREETYPE_BUILD_CMD, vc20xx=vc, WinXX=WinXX,
                                             config='Release' if VS2010 else 'LIB Release'))
 
-            os.environ['FREETYPE'] = src_path
-            subprocess.check_call([cmdfile], shell=True)
+            shutil.rmtree(str(Path(src_path, "objs")), ignore_errors=True)
+            subprocess.check_call([cmdfile], shell=True, cwd=src_path)
+            # Move to the corresponding Unix build path.
+            Path(src_path, "objs/.libs").mkdir()
+            # Be robust against change of FreeType version.
+            lib_path, = (Path(src_path, "objs", vc20xx, xXX)
+                         .glob("freetype*.lib"))
+            shutil.copy2(
+                str(lib_path),
+                str(Path(src_path, "objs", ".libs", "libfreetype.lib"))
```

Comment:
I would not use separate args if the dirs are explicit strings, i.e. just write `Path(src_path, "objs/.libs/libfreetype.lib")` but that's a matter of taste.